frontend/src/console/components/ThemeButtons/ThemeButtons.tsx

```diff
@@ -0,0 +1,163 @@
+    }
+    console.log(themeData);
+
+    const tooltip = (
+      <Tooltip id="redeployment-status">
+        <WrappedMessage messages={messages} id={`themePrimaryButtonHelp`} />
+      </Tooltip>
+    );
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        {themeData && themeData.version === 1 && (
+          <div>
+            <Row>
+              <Col md={9}>
+                <WrappedMessage messages={messages} id="themePrimaryButton" />
+                <OverlayTrigger placement="right" overlay={tooltip}>
+                  <div className="info-icon">
+                    <i className="fas fa-info-circle"/>
+                  </div>
+                </OverlayTrigger>
+              </Col>
+              <Col md={3}>
+                <CustomizableButton
+                  initialBackgroundColor={themeData.btnPrimaryColor}
+                  initialTextColor={themeData.btnPrimaryBg}
+                  initialBorderBlockColor={themeData.btnPrimaryBorderColor}
+                  initialHoverBackgroundColor={themeData.btnPrimaryHoverColor}
+                  initialHoverTextColor={themeData.btnPrimaryHoverBg}
+                  initialHoverBorderBlockColor={themeData.btnPrimaryHoverBorderColor}
+                />
+              </Col>
+            </Row>
+
+            <CollapseEditArea initialExpanded={true}>
+              <Container className="theme-colors-and-prevew-container">
+                <Row><h2>Active styles</h2></Row>
+                <Row>
+                  <Col>
+                    <ColorInputField
+                      fieldName="btnPrimaryBg"
+                      initialValue={themeData.btnPrimaryBg}
+                      onChange={this.onChangeColor}
+                      messages={messages}
+                      loading={instance.loading.includes('draftThemeConfig')}
+                      hideTooltip={true}
+                    />
+                  </Col>
+                  <Col>
+                    <ColorInputField
+                      fieldName="btnPrimaryColor"
+                      initialValue={themeData.btnPrimaryColor}
+                      onChange={this.onChangeColor}
+                      messages={messages}
+                      loading={instance.loading.includes('draftThemeConfig')}
+                      hideTooltip={true}
+                    />
+                  </Col>
+                  <Col>
+                    <ColorInputField
+                      fieldName="btnPrimaryBorderColor"
+                      initialValue={themeData.btnPrimaryBorderColor}
+                      onChange={this.onChangeColor}
+                      messages={messages}
+                      loading={instance.loading.includes('draftThemeConfig')}
+                      hideTooltip={true}
+                    />
+                  </Col>
+                </Row>
+              </Container>
+            </CollapseEditArea>
+
+            <CollapseEditArea initialExpanded={false}>
```

Comment:
@Agrendalath You just need 1 `CollapseEditArea` for each set of buttons.

Reply:
Right, good catch :+1: